src/test/java/org/mule/extension/validation/AggregateValidationTestCase.java

```diff
@@ -60,7 +60,7 @@ public void nonValidationErrorInsideAll() throws Exception {
 
   @Test
   public void nonValidationErrorMixedWithValidationErrorsInsideAll() throws Exception {
-    expected.expectErrorType(VALIDATION_NAMESPACE, MULTIPLE_ERROR);
+    expected.expectErrorType("MULE", EXPRESSION.getType());
```

Comment:
"MULE" looks like it should be a constant. Any reason why not to use a constant?

Reply:
we create the constant MULE_NAMESPACE and replace the "MULE" with it